app/db/shopit.sql

```diff
@@ -0,0 +1,40 @@
+USE shopit;
+
+DROP TABLE IF EXISTS user;
+CREATE TABLE user
+(
+    id                  varchar(255) NOT NULl,
+    username            varchar(20) NOT NULL,    # main email address serve as username
```

Comment:
Also is 20 too short for email address? It is arbitrary but we can at lease make it 60?
https://stackoverflow.com/questions/1305327/emailaddress-field-length-in-database